backend/main.py

```diff
@@ -66,6 +67,12 @@ def ValidateInputs(present, valid):
     elif a == 'missionId':
       if not firebase.get('/missions/%s/name' % data, None):
         raise InvalidInputError('Mission %s not found.' % data)
+    elif a == 'chatRoomId':
+      if not firebase.get('/chatRooms/%s/name' % data, None):
+        raise InvalidInputError('Chat room %s not found.' % data)
+    elif a == 'allegianceFilter':
+      if data not in ('horde', 'resistance', 'none'):
```

Comment:
It's easy enough to put it in the constants file or even into the database. Hard to say where else it will show up.